install_files/ansible-base/inventory-dynamic

```diff
@@ -131,9 +130,12 @@ def lookup_host_distro_id():
     Returns output of `lsb_release --id --short`, to check
     whether host machine is Tails.
     """
-    cmd = ["lsb_release", "--id", "--short"]
-    p = subprocess.check_output(cmd).decode("utf-8")
-    distro_id = p.rstrip()
+    with open("/etc/os-release") as f:
```

Comment:
In this case, `lookup_host_distro_id` is only ever used in the next function below, `host_is_tails`. So I think you can merge the two functions and just copy the same code from the earlier `is_tails` in admin/boostrap.py.